src/net35/Radical.Windows/Presentation/Behaviors/ListView Behaviors/ScrollIntoViewForListView.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Windows.Controls;
+
+namespace Topics.Radical.Windows.Behaviors
+{
+    public class ScrollIntoViewForListView : RadicalBehavior<ListView>
+    {
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+            this.AssociatedObject.SelectionChanged += AssociatedObject_SelectionChanged;
+        }
+
+        void AssociatedObject_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if (sender is ListView)
+            {
+                ListView listView = (sender as ListView);
```

Comment:
can be a direct cast:
```
var listView = (ListView)sender;
```

Or you can also use Radical `As<>` extension:

```
sender.As<ListView>( listView => {
   //this will invoked only if the cast succeeds
});
```